Morphic/Morphic.pier

```diff
@@ -323,21 +330,21 @@ all crossMorphs answer ==true== to the ==handlesMouseDown:== message.
 
 @@todo Add this method to ==CrossMorph==:
 
-[[[caption=Declaring that ==CrossMorph== will react to mouse clicks.
-CrossMorph>>>handlesMouseDown: anEvent
-	^true
+[[[language=smalltalk|caption=Declaring that ==CrossMorph== will react to mouse clicks|label=scr:mouseEvent
+CrossMorph>>#handlesMouseDown: anEvent
```

Comment:
I'm not sure we should switch our script convention here. (Or, if this is the right notation, we should go back and fix all other method definitions in the scripts).
I personally don't like the `>>>` notation that the book uses, but I understand it compiles into the special >> character.
So, I'm not necessarily arguing against `>>#`, but just pointing out we should be consistent.